README.md

```diff
@@ -88,7 +131,5 @@ curl http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar > /tmp/php-cs-fixer.
 php /tmp/php-cs-fixer.phar  fix -v --diff --using-cache=yes src/
```

Comment:
this should be in the ci instead...

Reply:
Not with `fix`, just the test, right? But there's no CI tests, currently

Reply:
oh right, needs to be test then. created #4